docs/plugins/first_plugin.md

```diff
@@ -21,6 +21,22 @@ recommended**). If you are new to virtual environments, we recommend
 [installing miniconda][miniconda] and [creating a new environment with
 python][python_env].
 
+Even though your plugin does not need to directly depend on the `napari`
+package, and
```

Comment:
```suggestion
Even though plugins don't necessarily need to list `napari` as a direct dependency, and
```

Reply:
(mostly suggesting the "necessarily", since we later go on to demonstrate a plugin that _does_ need to list napari)